### ISSUE TITLE
Align string concatenations vertically

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#2127]
In case a string concatenation spans multiple lines, we should issue a warning in case it is not aligned vertically. The code fix should align it vertically.

Example
```c#
string s = "some text"
    + "some other text"
    + "some final text";
```

Should be aligned to:

```c#
string s = "some text"
         + "some other text"
         + "some final text";
```

The alignment helps reading.
